peer/record.go

```diff
@@ -0,0 +1,188 @@
+package peer
+
+import (
+	"errors"
+
+	"github.com/gogo/protobuf/proto"
```

Comment:
Group all libp2p imports first, then all third-party libraries in another group.

Reply:
@yusefnapora You may have missed this comment?